communication/socket/tcp/send/wsaioctl_transmitpackets_callbackfunc.yml

```diff
@@ -0,0 +1,28 @@
+rule:
+  meta:
+    name: obtains TransmitPackets callback function via WSAIoctl  
```

Comment:
the filename should match the rule name, lowercase and using dashes instead of spaces